src/popup/components/IconWrapper.vue

```diff
@@ -12,16 +12,16 @@
       class="icon-wrapper-icon"
       :class="{'icon-wrapper-fill': isFull}"
     />
-    <ProtocolIcon
+    <AssetIcon
       v-else-if="protocolIcon"
-      :protocol="protocolIcon"
+      :contract-id="protocolIcon"
```

Comment:
`protocolIcon` is of type `Protocol` so its not matching the `contractId` type. I mean it kind of matches because we use the same values for the protocol names and the `contractId` of the coins. But this is a coincidence and might change in the future.

Reply:
I changed it so that we now use the `contract-id` of the protocols